crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -156,25 +166,15 @@ impl CallContractCommand {
 		}
 
 		// Resolve path.
-		let contract_path = match self.path.as_ref() {
-			None => {
-				let path = Some(PathBuf::from("./"));
-				if has_contract_been_built(path.as_deref()) {
```

Comment:
This was causing a panic if you run `pop call contract` outside a contract folder.